pkg/nvpci/nvpci.go

```diff
@@ -521,6 +528,31 @@ func getDriver(devicePath string) (string, error) {
 	return "", err
 }
 
+// /sys/bus/pci/devices/0000:df:00.0/vfio-dev/vfio0.
+func getIOMMUFD(devicePath string) (int, error) {
+	vfioDevDir := filepath.Join(devicePath, "vfio-dev")
+	// Read the directory; expect exactly one entry
+	entries, err := os.ReadDir(vfioDevDir)
+	switch {
+	case os.IsNotExist(err):
+		return -1, nil
+	case err == nil:
+		if len(entries) == 0 {
```

Comment:
Should we check for the `vfio` prefix in the entries? Does using `filepath.Glob` make sense here if we're expecting a particular pattern?